components/renku_data_services/crc/orm.py

```diff
@@ -133,6 +136,24 @@ def dump(self, matching_criteria: models.ResourceClass | None = None) -> models.
         )
 
 
+class ClusterORM(BaseORM):
+    """Cluster definition."""
+
+    __tablename__ = "clusters"
+    id: Mapped[ULID] = mapped_column("id", ULIDType, primary_key=True, default_factory=lambda: str(ULID()), init=False)
+    name: Mapped[str] = mapped_column(String(40), unique=True, index=True)
```

Comment:
question: Do we care if this is unique? This `name` field is only for the benefit of users. We have the primary key to identify clusters with - that is unique. But I think this does not have to be, right?

Reply:
Leave as is.